### ISSUE TITLE
Release command may interpret version as a number

### ISSUE THREAD

**Issue by jkrems** [Microsoft/code-push#286]
Given the following command:

```bash
code-push release FooBar scripts 8.10
```

The CLI will treat the last argument as "8.1" because the implicit number conversion kicks in. I've seen that most flags already set the type to string so I assume it only affects a few commands. But maybe it would still be a good idea to either require a `v` prefix ("v8.10") or an explicit ".0" to prevent future bugs connected to this..?

**Comment by silhouettes** [comment]
Thanks for catching this! Fortunately it only seems to apply to the `release` command, and not the `release-react` and `release-cordova` commands, which might explain why we haven't seen it yet. I've just put up a fix for the root cause (the number conversion), but let me know if you'd still like to see the other changes you mentioned!

**Comment by jkrems** [comment]
Thanks!

> I've just put up a fix for the root cause (the number conversion), but let me know if you'd still like to see the other changes you mentioned!

Nah, the rest of the issue was just me being paranoid about it. 👍

**Comment by geof90** [comment]
While @silhouettes 's change is going in and before we release to NPM, could you try wrapping the value in quotes as a temporary workaround? So `code-push release FooBar scripts "8.10"`

**Comment by jkrems** [comment]
Since according to the docs "8.10" and "8.10.0" are treated the same, we're just using "8.10.0". I *think* bash quoting doesn't make it into the node process:

```
> node -p 'process.argv' -- foo "bar"
[ '/Users/jkrems/.nvm/versions/node/v4.4.7/bin/node',
  'foo',
  'bar' ]
```

**Comment by silhouettes** [comment]
This was fixed with release `1.12.3-beta`, thanks for filing this!
